_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Until now, NewPipe had a main, pop-up and background -player. All three had separate code and playing queues. Switching from one player to another meant the old player and its queue closed itself, while a new player and queue was created. That sounds complicated and cumbersome, because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start anew instead of carrying on from where you left off. Losing your (possibly carefully built) queue was the bonus.
 
-Well, no more! The buttons in the player drop down menu have been removed and the action buttons below the thumbnail work like they're supposed to, so now you're at no risk whatsoever of losing your playing position or queue. Whew!
+No more! There are no buttons in the player drop-down menu anymore, and the action buttons below the thumbnail now only do the job of being action buttons.
+Everyone keeps their position _in the queue_. The utopia is here.
 
-This was made possible by rewriting the code so that all players run on one common service. When you switch from one player to another, this service remains running, preserving the stream data, and the new player just picks up from where the old one left off. Everything remains the same, including the queue. Ah, serendipity.
+An equally complicated and cumbersome code rewrite lets all views use one common service. Switch from one player to another, and the service is at your service preserving the stream data, changing only the view. If anything your blood-pressure could drop _too low_. Ah, serendipity.
 
 ### Video details page
 
-A more unified experience is not the only feature this change brought you. There is one which is immediately noticeable once you open a video in the new version: the video details page. Earlier, when you opened the video details, you were shown a thumbnail, video description, comments, etc. Tapping the thumbnail would open the player in full-screen, hiding the rest of these elements. Now, when you're in portrait mode, the video plays right in the video details page - just like YouTube! There is even a new 'Autoplay' setting which plays the video automatically when you open video details - also just like YouTube! This setting will be turned on by default for Wi-Fi. Of course, you can turn it off if you prefer the behaviour of the previous versions.
```

Comment:
>A more unified experience is not the only feature this change brought you. There is one which is

All of that is nothing. One sentence more into what is a garden-path sentence the reader knows we are talking about the video details page. Another mechanical explanation, OK. Where it is either talking directly to the user, there is no flow onto this. Don't say it is like YouTube, because that is unprofessional, and legal fodder for their legal team. NewPipe specifically doesn't do that anywhere I can help it. "This setting will be turned on by default" isn't good language. Who knows what the behavior of the previous versions is. Not people downloading for the first time.

Reply:
>the reader knows we are talking about the video details page

Then does it matter? This is again a clash of writing styles. You prefer being straightforward and getting to the point immediately. The tone of the blog post is more informal.

> there is no flow onto this

Explain?

> legal fodder for their legal team

@TheAssassin Your input?

> Who knows what the behavior of the previous versions is. -Not people downloading for the first time.

Which will be cleared up the moment they read the very next sentence.

Reply:
Read trademark law. Where isn't NewPipe welcoming a lawsuit?

If you don't muddy the waters, you don't need to clear up. There is no saving grace there.
Here is how it works: There is a setting, refers to earlier sentence, between "just like youtube", even earlier that is a referenced "setting" (bad repetition", redundant "which" "plays the video automatically when you open video details". However "in portrait mode" OK, That setting is on. "by default" isn't needed then. Why not just explain what happens right there and then? "Of course", what? 
"you" not needed "can turn it off" what is it here? "if you prefer" second you, "the behaviour of" waiting "the previous versions." So "or it can be turned off." that is all I read there. In 4 multi-stage sentences all interlinked. That is not a writing style, that is a mistake.